sklearn/decomposition/pca.py

```diff
@@ -167,6 +165,9 @@ class PCA(_BasePCA):
             dimension of the data, then the more efficient 'randomized'
             method is enabled. Otherwise the exact full SVD is computed and
             optionally truncated afterwards.
+
+            in case sparse data is used, 'randomized' is used as this is the
```

Comment:
```suggestion
            In case sparse data is used, 'randomized' is used as this is the
```

Reply:
It was kind of weird for only this paragraph to start with a capitalized letter, so I capitalized all the paragraphs for `svd_solver`. IMO this is much better, but it is different from what SVD has.